src/GprTool/NuGetUtilities.cs

```diff
@@ -36,10 +34,8 @@ public static bool ShouldRewriteNupkg(string nupkgPath, string repositoryUrl, Nu
             }
 
             var nuspecRepositoryUrl = repositoryXElement.Attribute("url")?.Value;
-            var nuspecRepositoryType = repositoryXElement.Attribute("type")?.Value;
-
-            return !string.Equals(repositoryUrl, nuspecRepositoryUrl, StringComparison.Ordinal) 
-                   || !string.Equals("git", nuspecRepositoryType, StringComparison.Ordinal);
+            
+            return !string.Equals(repositoryUrl, nuspecRepositoryUrl, StringComparison.Ordinal);
```

Comment:
We should probably ignore case here. Owners/repos often have uppercase chars, which are ignored when people type a URL by hand.

```suggestion
            return !string.Equals(repositoryUrl, nuspecRepositoryUrl, StringComparison.OrdinalIgnoreCase);
```